### ISSUE TITLE
Creator's update: start menu,  native input  NVDA report English string

### ISSUE THREAD

**Issue by surfer0627** [nvaccess/nvda#7174]
This case occurs in 		V_1703:
Steps to reproduce:
1. Press windows key to open start menu.
2. Type string "2l" (Chinese = "ㄉㄠ")
  * speech: 2ㄉ lㄠ
  * expected: ㄉㄠ

**Comment by surfer0627** [comment]
This case also occurs. When user press windows key + i to open settings menu. Switch to native input, and type Chinese string.

**Comment by surfer0627** [comment]
@ehollig, yes, I can reproduce this case in 1803.

**Comment by Adriani90** [comment]
@surfer0627 how is this now with NVDA 2020.1 RC1 and Windows 10 1909 Update?

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since many years, and I cannot reproduce it with NVDA 2023.1 on Windows 10 or Windows 11 last update. Closing as works for me. Please comment on it if you are still having this issue and we can reopen.

**Comment by keyang556** [comment]
@Adriani90 @seanbudd This issue reproducible in win11 latest version and 2026.3 alpha, please re-open it, thanks.

**Comment by keyang556** [comment]
Beside start menu and windows settings, many UWP apps reproduce this.

**Comment by seanbudd** [comment]
Can we please get a new issue? The original issue has almost no information.

**Comment by seanbudd** [comment]
Can you confirm if you have automatic language switching enabled? Can you upload a debug log in a new issue?
